README.md

```diff
@@ -31,35 +39,58 @@ For example:
++------------------------------------------+---------------------+-----------------------+
+|                   HASH                   |    AUTHOR EMAIL     |      AUTHOR NAME      |
++------------------------------------------+---------------------+-----------------------+
+| 003dc36e0067b25333cb5d3a5ccc31fd028a1c83 | user1@test.io       | Santiago M. Mola      |
+| 01ace9e4d144aaeb50eb630fed993375609bcf55 | user2@test.io       | Antonio Navarro Perez |
++------------------------------------------+---------------------+-----------------------+
+```
+
+Also you can use the interactive shell like you usually do to explore tables in postgreSQL per example:
+
+```bash
+$ gitql shell
+
+           gitQL SHELL
+           -----------
+You must end your queries with ';'
+
+!> SELECT hash, author_email, author_name FROM commits LIMIT 2;
+
+--> Executing query: SELECT hash, author_email, author_name FROM commits LIMIT 2;
+
++------------------------------------------+---------------------+-----------------------+
+|                   HASH                   |    AUTHOR EMAIL     |      AUTHOR NAME      |
++------------------------------------------+---------------------+-----------------------+
+| 003dc36e0067b25333cb5d3a5ccc31fd028a1c83 | user1@test.io       | Santiago M. Mola      |
+| 01ace9e4d144aaeb50eb630fed993375609bcf55 | user2@test.io       | Antonio Navarro Perez |
++------------------------------------------+---------------------+-----------------------+
+!>  
 ```
 
 ## Tables
 
 gitql exposes the following tables:
 
-* commits (hash, author_name, author_email, author_time, comitter_name, comitter_email, comitter_time, message)
-* blobs (hash, size)
-* references (hash, name, is_branch, is_note, is_remote, is_tag, target)
-* tags (hash, name, tagger_email, tagger_name, tagger_when, message, target)
-* tree_entries (tree_hash, entry_hash, mode, name)
+|     Name     |                                               Columns                                               |
+|:------------:|:---------------------------------------------------------------------------------------------------:|
+|    commits   | hash, author_name, author_email, author_time, comitter_name, comitter_email, comitter_time, message |
+|     blobs    | hash, size                                                                                          |
+|  references  | hash,hash, name, is_branch, is_note, is_remote, is_tag, target                                      |
+|     tags     | hash, name, tagger_email, tagger_name, tagger_when, message, target                                 |
+| tree_entries | tree_hash, entry_hash, mode, name                                                                   |
 
 ## SQL syntax
 
-gitql supports a subset of the SQL standard, currently including:
+We are adding continuously more functionality to gitql. Actually we supports a subset of the SQL standard, currently including:
```

Comment:
We are adding continously -> We are continuously adding
Actually we supports -> We support